test/net/sourceforge/kolmafia/objectpool/ConcoctionTest.java

```diff
@@ -22,70 +24,76 @@ public class ConcoctionTest {
   // properly handled by the compareTo.
   @Test
   public void itShouldSortUsables() {
-    LockableListModel<Concoction> usableList = ConcoctionDatabase.getUsables();
-    int thing = usableList.size();
-    usableList.sort();
-    assertEquals(usableList.size(), thing);
+    var useables = ConcoctionDatabase.getUsables();
+    int sizeBeforeSort = useables.size();
+    useables.sort();
+    assertThat(useables.size(), equalTo(sizeBeforeSort));
+  }
+
+  @Test
+  public void steelOrgansAreSortedToTheTop() {
+    var steelMargarita = ConcoctionPool.get(ItemPool.STEEL_LIVER);
+    var nonSteelOrgan = ConcoctionPool.get(ItemPool.PERFECT_NEGRONI);
+    assertThat(nonSteelOrgan, notNullValue());
```

Comment:
Why is this assertion useful?  If ConcoctionPool can return a null then shouldn't the steel margarita be checked also?

Reply:
Good point - it is a hangover from how I did this before.